features/app/components/AbstractApp.js

```diff
@@ -22,6 +31,30 @@ export class AbstractApp extends Component {
     }
 
     /**
+     * Init lib-jitsi-meet and create local participant when component is going
+     * to be mounted.
+     *
+     * @inheritdoc
+     */
+    componentWillMount() {
```

Comment:
Will React call componentWillUnmount if componentWillMount fails? Or if the constructor fails (though I guess it will not)? I'm asking because componentWillUnmount in this case is supposed to revert the effects of what you currently have in componentWillMount.